sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -1,3 +1,16 @@
+namespace Azure
+{
+    public partial class StreamingResponse<T> : System.Collections.Generic.IAsyncEnumerable<T>, System.IDisposable
```

Comment:
Open design question: should `StreamingResponse<T>`, either automatically or via opt-in, cache enumerated items for subsequent reuse?

E.g., as written here, if I do `await foreach (Foo foo in fooResponse)` and the fooResponse was enumerating from a content stream, making that call *again* will fail, as the stream no longer contains the data.

Reply:
After a bit of arch review, we'll leave this exploration a bit further in the future (as we consider the class for a more central concept in Azure Core) -- it's currently "easy enough" to do this as a consumer if desired and we can evaluate the right approach for automatic caching later on.